sources/packages/forms/src/form-definitions/sfaa2022-23.json

```diff
@@ -31157,7 +31159,7 @@
                   "label": "Please estimate your weekly transportation costs.",
                   "key": "additionalTransportCost",
                   "placeholder": "",
-                  "prefix": "",
+                  "prefix": "$",
```

Comment:
We are missing the km suffix here and in the next PY form as well. 

Please ensure that the forms are consistent.